### ISSUE TITLE
UAX 14 not strictly required by css-text

### ISSUE THREAD

**Issue by frivoal** [web-platform-tests/wpt#46493]
I suspect the following two tests ought to be marked as "should" tests rather than the default must, as per https://web-platform-tests.org/writing-tests/css-metadata.html#requirement-flags, For the reasons covered in the Note at the end of https://drafts.csswg.org/css-text-4/#line-breaking

https://github.com/web-platform-tests/wpt/blob/master/css/css-text/line-breaking/line-breaking-028.html
https://github.com/web-platform-tests/wpt/blob/master/css/css-text/line-breaking/line-breaking-029.html

cc: @makotokato, @jfkthame (as authors/reviewers)

**Comment by jfkthame** [comment]
I haven't tried to check extensively, but I suspect there are probably quite a few tests where this is the case -- they assume line-breaking behavior that is not strictly required by the spec, but in practice is what everyone expects. Unless there's an implementation that is actually wanting to deviate here, without being called out as non-compliant, I'm not sure there's much reason to soften the tests to use "should". What would the practical results of this be -- how do "should" tests (whether passing or failing) show up on wpt.fyi and other forms of reporting?

**Comment by frivoal** [comment]
I don't believe wpt currently distinguishes. The old csswg test system did distinguish: when reporting results, it would separate "must" tests from "should" ones, which is useful when trying to progress a spec, as must failures are a sign that either implementations or the spec need fixing, while should failures can be allowed to persist. I hope that wpt will eventually gain this ability

**Comment by makotokato** [comment]
all css-text/i18n/* tests uses UAX14 rules. So I guess that  it should apply "SHOULD" too.

**Comment by frivoal** [comment]
UAX14 in general isn't mandated by CSS, and is indeed a good match for SHOULD, but there are some parts of it which are specifically mandated, so they're not *all* SHOULDs
